docs/.eleventy.js

```diff
@@ -76,6 +77,9 @@ module.exports = function (eleventyConfig) {
   //   }
   // });
 
+  eleventyConfig.on('beforeWatch', () => {
```

Comment:
>  It's a little awkward though (the implementation of running the command in the eleventy config file and the fact it triggers another watcher build) not sure how useful it is.

could you try ignoring the source markdown files via https://www.11ty.dev/docs/ignores/ ?

Reply:
oh wait, but then that wouldn't trigger watch in the first place. hmm... 

The unnecessary double-build isn't terrible and definitely an improvement on what we have, feels like this can be fine for now